README.md

```diff
@@ -83,3 +83,20 @@ MY_SPECIAL_VAR:
 ```
 
 At runtime the value of `MY_SPECIAL_VAR` will be replaced with the value from parameter store
+
+Adding a new secret environment variable to databox and production
+---------------------------------------------
+You may want to add a new enviroment variable to your databox for testing and then commit this to production when ready merging.
+
+To do this:\
+1. Navigate to the beetle folder
+2. Run ```envars add VAR_NAME=value -s```
+3. One the above line is ran, the terminal will print the secret to the variable like below:
+
+```
+default: !secret |-
+      AQICAHiOrkzo4U0Qe3l3gBab8dyFfpnlamupaQ8EuHEULsA6uwGWjI8fnd7xt
+```
+
+4. Copy and paste this into the envars.yml file and include it in your PR.
```

Comment:
the `envars` command add the new var to the file automatically, no copy and paste required